src/simulation_lifecycle/convert.c

```diff
@@ -111,6 +122,8 @@ int convert_results(char *input, char *output, cJSON *visualization) {
 
         /* Comparing the 46th first characters of the third line
          * with "[cadmium::celldevs::cell_ports_def<std::string"*/
+        // TODO (Román) Careful with this!!! It is highly dependent on the compiler (see the macro definitions)
+        // TODO (Román) Can we just remove this comparison? it is too narrow, in my opinion
         if (strncmp(line, VALID_FORMAT, strlen(VALID_FORMAT)) != 0) {
```

Comment:
Here, it fails for Mac and Ubuntu. Tests passed because we were using a log file generated in a Windows machine. I suggest that we remove this assertion. What do you think, @francoiseblanc ?